app/post_metadata.go

```diff
@@ -209,6 +214,13 @@ func (a *App) getEmbedForPost(post *model.Post, firstLink string, isNewPost bool
 		}, nil
 	}
 
+	if _, ok := post.GetProps()["boards"]; ok {
+		return &model.PostEmbed{
```

Comment:
For message attachment embeds, we don't attach any data to the embed itself because the client knows to look at the props to get that data. Would it make sense to do that with boards embeds as well to avoid duplicating that data?